src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -89,6 +90,30 @@ module sonic-portchannel {
 
 		} /* end of container PORTCHANNEL */
 
+		container PORTCHANNEL_MEMBER {
+
+			description "PORTCHANNEL_MEMBER part of config_db.json";
+
+			list PORTCHANNEL_MEMBER_LIST {
+
+				key "portchannel_name port";
+
+				leaf portchannel_name {
```

Comment:
Can we change it to 'name'

Reply:
do not understand the comment. is this for me or is this for you?

Reply:
@lguohan This was for me, in YANG WG, there was a suggestion to keep leaf as 'name' instead of 'portchannel_name'. So I noted it down here.